flake.nix

```diff
@@ -1,98 +1,30 @@
 {
   description = "agenix-cli";
 
-  inputs.nixpkgs.url = "github:nixos/nixpkgs/nixos-21.11";
-  inputs.crate2nix = { url = "github:kolloch/crate2nix"; flake = false; };
-  inputs.mozilla = { url = "github:mozilla/nixpkgs-mozilla"; flake = false; };
+  inputs.nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
+  inputs.flake-utils.url = "github:numtide/flake-utils";
 
-  outputs =
-    { self
-    , nixpkgs
-    , mozilla
-    , ...
-    } @ inputs:
+  outputs = { self, nixpkgs, flake-utils }: flake-utils.lib.eachDefaultSystem (system:
```

Comment:
Nitpick, but would you mind moving the `flake-utils` stuff to its own line, following the inputs?